File: backup_before_industrialisation_20250626/backend/metier_generator/templates/threed/hooks/lifecycle/lifecycle_hooks.js
// Hooks lifecycle threed (Node.js)
function beforeLifecycleEvent(event, data) {
  // Logique avant événement de cycle de vie
  return { event, data };
}
function afterLifecycleEvent(event, data) {
  // Logique après événement de cycle de vie
  return { event, data };
}
module.exports = { beforeLifecycleEvent, afterLifecycleEvent };

File: backup_before_industrialisation_20250626/backend/metier_generator/templates/threed/hooks/lifecycle/lifecycle_hooks.py
"""
Hooks lifecycle threed (Python)
"""


def before_lifecycle_event(event, data):
    # Logique avant événement de cycle de vie
    return {"event": event, "data": data}


def after_lifecycle_event(event, data):
    # Logique après événement de cycle de vie
    return {"event": event, "data": data}
